app/src/main/java/org/listenbrainz/android/presentation/features/yim/navigation/YimShareables.kt

```diff
@@ -0,0 +1,10 @@
+package org.listenbrainz.android.presentation.features.yim.navigation
+
+enum class YimShareables(val code: String) {
```

Comment:
We could just name it as YimShareable and omit the s.
Ik its a small thing but helps maintain the sanctity of the code.

Reply:
Also, what is "code" here?

Reply:
XD the android studio suggested me this but i went this way forcefully xD. Will do

Reply:
> Also, what is "code" here?

The string that the api accepts to get response

Reply:
Right, I was suggesting an enum of the format

```kotlin
enum class YimShareable {
    TRACKS,
    STATISTICS,
    ARTISTS,
    ALBUMS,
    DISCOVERIES,
    UNDISCOVERED
}
```
But upon investigating further, I understood why you need the code, in order to pass it to the api. Sounds okay to me but I would have rather had a switch case while processing the saveSharableImage function by passing the Shareable type as the YimShareable. Both approaches sound good to me tbh. Just wanted to share where I was coming from.

Reply:
Yea we could have a switch case but i think it'll result in same number of lines of code. Also ill capitalize the enums!